release-please-config.json

```diff
@@ -1,22 +1,24 @@
 {
   "bootstrap-sha": "7870a6ed7450d211ae822a281de45b6fc3df39c3",
-  "include-component-in-tag": true,
```

Comment:
This should work here, it is what we do in go.

Reply:
Even if I only want it for the `telemetry` package?

I'm aiming for:
- The base SDK releases as just v1.0.0, v1.1.0, etc.
- The telemetry releases as telemetry-v1.0.0, etc.

Let me know if that doesn't make sense

Reply:
It makes the default true, so then you make it false in the base SDK.

Then any future packages automatically should include the component in the tag. Versus having to be careful to set it on all new packages.

Reply:
I am not sure why it made a combined PR and didn't include the tag in the version headers.

I've not used that mixed configuration in a combined PR before.